app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageRecipientFieldsManager.kt

```diff
@@ -125,11 +124,12 @@ class NewMessageRecipientFieldsManager @Inject constructor(private val snackbarM
     fun setOnFocusChangedListeners() = with(binding) {
         val listener = View.OnFocusChangeListener { _, hasFocus -> if (hasFocus) fieldGotFocus(null) }
         subjectTextField.onFocusChangeListener = listener
-        bodyTextField.onFocusChangeListener = listener
+        editorWebView.onFocusChangeListener = listener
     }
 
     fun focusBodyField() {
-        binding.bodyTextField.showKeyboard()
+        // TODO: Make it so keyboard is kept open through configuration changes whenever the editor gets focused
```

Comment:
Is this already handled in another PR ?

Reply:
No. If we ever change that part, this could be an opportunity change this as well, but for the time being, we don't have the time to bother with this